core/src/main/scala/io/projectglow/transformers/normalizevariants/NormalizeVariantsTransformer.scala

```diff
@@ -17,72 +17,122 @@
 package io.projectglow.transformers.normalizevariants
 
 import io.projectglow.DataFrameTransformer
+import io.projectglow.common.VariantSchemas._
 import io.projectglow.common.logging.{HlsEventRecorder, HlsTagValues}
+import io.projectglow.functions.{expand_struct, normalize_variant}
+import io.projectglow.transformers.normalizevariants.VariantNormalizer._
+import io.projectglow.transformers.splitmultiallelics.VariantSplitter
 import io.projectglow.transformers.util.StringUtils
+
 import org.apache.spark.sql.DataFrame
-import io.projectglow.transformers.splitmultiallelics.VariantSplitter
+import org.apache.spark.sql.functions._
 
 /**
- * Implements DataFrameTransformer to transform the input DataFrame of varaints to an output
- * DataFrame of normalized variants (normalization is as defined in bcftools norm).
+ * Implements DataFrameTransformer to transform the input DataFrame of variants to an output
+ * DataFrame of normalized variants (normalization is as defined in vt normalize or bcftools norm).
+ *
+ * A path to reference genome containing .fasta, and .fai files must be provided
```

Comment:
Is this path to the fasta file or do a directory containing both?

Reply:
Clarified.